sdks/python/apache_beam/io/hdfs_integration_test/hdfs_integration_test.sh

```diff
@@ -35,6 +35,8 @@ cp -r ${ROOT_DIR}/model ${CONTEXT_DIR}/
 PROJECT_NAME=$(echo hdfs_IT-${BUILD_TAG:-non-jenkins})
 
 cd ${CONTEXT_DIR}
+# TODO: prune?
+docker network ls
```

Comment:
Maybe
docker network prune -f

Reply:
I first needed to prove my hunch, and I don't know of a way to run postcommits on jenkins without creating a PR.